dbt/include/teradata/macros/columns.sql

```diff
@@ -1,3 +1,8 @@
+/* This macro is used when there is mismatch in column types from source while incremental modelling.
+For e.g We had a incremental run already for a project and later some user came and played with column type of 
+one of the column of the source and changed it e.g from varchar(20) changed it to varchar(25)
+So to sync up the data types this macro comes into play.
+*/
```

Comment:
Sorry, I didn't mean to suggest that we should explain why dbt has this macro. IMO, we only should explain why we need to override the default implementation.

```suggestion
/* The macro had to be overridden for Teradata because original macro uses `column` keyword that is not supported nor needed in Teradata's `ALTER TABLE` syntax.
*/
```

Reply:
Sorry, I misinterpreted.
Sure, will change the comment and have added the testcase for the same.